assets/src/data/eventespresso/lists/entities.js

```diff
@@ -13,14 +15,19 @@ import { MODEL_NAMES } from '../../model';
  */
 export const createEntitySelectors = ( source ) => MODEL_NAMES.reduce(
 	( selectors, modelName ) => {
-		selectors[ getMethodName( modelName, '', 'get', true ) ] = (
+		const methodNameForGet = getMethodName( modelName, '', 'get', true );
+		selectors[ methodNameForGet ] = (
 			state,
 			queryString,
 		) => source.getEntities( state, modelName, queryString );
 		selectors[ getMethodName( modelName, '', 'isRequesting', true ) ] = (
 			state,
 			queryString,
-		) => source.isRequestingEntities( state, modelName, queryString );
+		) => isResolving(
+			REDUCER_KEY,
+			methodNameForGet,
+			queryString
+		);
```

Comment:
So I discovered this bug while testing things on this branch when I decided to try the new dynamically generated model selectors (`getAttendees` and `isRequestingAttendees`).  I realized that in the original iteration `isRequestingAttendees` would end up checking the resolution state for `getEntities` because it's wrapping a call to `isRequestingEntities`.  However the resolution state would be indexed on the `getAttendees` selector (which also has a different set of arguments).  Making this change here (and for the schema registry) fixed things.